server/aws-lsp-codewhisperer/src/language-server/netTransform/models.ts

```diff
@@ -22,6 +22,7 @@ export interface StartTransformRequest extends ExecuteCommandParams {
     ProjectMetadata: TransformProjectMetadata[]
     TransformNetStandardProjects: boolean
     EnableRazorViewTransform: boolean
+    EnableWebFormsToBlazorTransform: boolean
```

Comment:
is this should be optional ? if not release in toolkits is that will cause issue?